documentation/src/main/asciidoc/topics/proc_creating_caches_cli.adoc

```diff
@@ -0,0 +1,52 @@
+[id='create_cache_cli']
+= Creating Caches with the {brandname} Command Line Interface (CLI)
+Use the {brandname} CLI to add caches from templates or configuration files in
+XML or JSON format.
+
+.Prerequisites
+
+Add {brandname} credentials and start at least one {brandname} server instance.
+
+.Procedure
+
+. Create a CLI connection to {brandname}.
+. Add cache definitions with the [command]`create cache` command.
++
+* Add a cache definition from an XML or JSON file with the [command]`--file` option.
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> create cache --file=configuration.xml mycache
+----
++
+[TIP]
```

Comment:
shouldn't the "tip" be in the bullet below? I find it weird to read about the "--file" option with a tip about a "--template" option,